dev/Dockerfile

```diff
@@ -13,51 +13,45 @@ RUN apt-get update && apt-get install -y \
     git \
     libgsl-dev \
     make \
-    unzip \
-    vim \
-    wget && \
+    unzip && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # corsika and sim_telarray
 RUN mkdir sim_telarray
 COPY corsika7.7_simtelarray.tar.gz sim_telarray
 RUN cd sim_telarray && \
-    tar -xvzf corsika7.7_simtelarray.tar.gz && \
+    tar -xvf corsika7.7_simtelarray.tar.gz && \
     ./build_all prod5 qgs2 gsl && \
     find . -name "*.tar.gz" -exec rm -f {} \; && \
     cd ..
 
-RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$(uname -p).sh -O ~/miniconda.sh && \
-    /bin/bash ~/miniconda.sh -b -p /workdir/conda && \
-    rm ~/miniconda.sh && \
-    /workdir/conda/bin/conda clean -tipy
-
-RUN wget --quiet https://raw.githubusercontent.com/gammasim/simtools/master/environment.yml
-
 From ubuntu:22.10
 WORKDIR /workdir
 ENV DEBIAN_FRONTEND=noninteractive
 COPY --from=build_image /workdir/sim_telarray/ /workdir/sim_telarray/
-COPY --from=build_image /workdir/conda /workdir/conda
-COPY --from=build_image /workdir/environment.yml /workdir/environment.yml
 
 RUN apt-get update && apt-get install -y \
+    bc \
     bzip2 \
     gfortran \
     libgsl-dev \
-    bc \
-    unzip && \
+    python3-pip \
+    unzip \
```

Comment:
Are you sure we still need unzip here?

Reply:
No, we don't need it. I've removed it now.